### PR TITLE
Timetable display view is back

### PR DESCRIPTION
**Pull request by OmeGak**

### CODE REVIEW

File: indico/modules/events/timetable/controllers.py [hunk 1]
```diff
@@ -109,3 +113,17 @@ def _process_PATCH(self):
         if updates:
             update_timetable_entry(self.timetable_entry, updates)
         return jsonify()
+
+
+class RHTimetable(RHConferenceBaseDisplay):
+    def _checkParams(self, params):
+        RHConferenceBaseDisplay._checkParams(self, params)
+        self.event = self._conf
```
Comment (ThiefMaster) [inline indico/modules/events/timetable/controllers.py]: I'd rather keep using `self._conf` for this. I know we used this assignment quite a lot when we didn't have an `Event` object but now it's just really confusing

File: indico/modules/events/timetable/controllers.py [hunk 2]
```diff
@@ -109,3 +113,17 @@ def _process_PATCH(self):
...
+        self.layout = request.args.get('ttLyt', '')
```
Comment (ThiefMaster) [inline indico/modules/events/timetable/controllers.py]: Would it be much work to change this awful name (and just keep it as a fallback in case people link to it)?

    self.layout = request.args.get('layout', request.args.get('ttLyt', ''))

File: indico/modules/events/timetable/util.py [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+from collections import defaultdict
+
+from flask import session
+
+from indico.modules.events.timetable.models.entries import TimetableEntry, TimetableEntryType
+from indico.util.date_time import iterdays
+
+
+def serialize_timetable(event, days=None, mgmtMode=False, useAttrCache=False, hideWeekends=False):
```
Comment (ThiefMaster) [inline indico/modules/events/timetable/util.py]: useAttrCache is unused - and you could already snakify the other ones (`hide_weekends`, `management`)?

File: indico/web/assets/bundles.py [hunk 4]
```diff
@@ -406,8 +406,8 @@ def rjs_bundle(name, *files):
     'contributions': rjs_bundle('modules_contributions',
                                 *namespace('js/indico/modules/contributions', 'common.js', 'userlists.js')),
     'timetable': rjs_bundle('modules_timetable',
-                            *namespace('js/indico/modules/timetable/timetable', 'Base.js', 'DragAndDrop.js', 'Draw.js',
-                                       'Filter.js', 'Layout.js', 'Management.js', 'Undo.js')),
+                            *namespace('js/indico/modules/timetable/timetable', 'Filter.js', 'Layout.js', 'Undo.js',
```
Comment (ThiefMaster) [inline indico/web/assets/bundles.py]: What's the reason for this change? Seems wrong that `Base.js` is not the first one...

Comment (OmeGak) [inline indico/web/assets/bundles.py]: Import order.

File: indico/modules/events/timetable/templates/_timetable.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+            <div class="text" style="padding-top: 200px;">
+                {% trans %}Building timetable...{% endtrans %}
+            </div>
+        </div>
+        <div class="clearfix"></div>
+    </div>
+
+    <script>
+        $(window).on('load', function() {
+            var timetableLayout = {{ timetable_layout|tojson }} || null;
```
Comment (ThiefMaster) [inline indico/modules/events/timetable/templates/_timetable.html]: Why not just default to `None` in the Python code? That'll be converted to `null` in tojson
